addons/l10n_ar_withholding/views/account_payment_view.xml

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="view_account_payment_form" model="ir.ui.view">
+        <field name="name">account.payment.form.inherited</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <group position="inside">
+                <group name="group4" colspan="2" invisible="country_code != 'AR'">
+                    <field name="withholding_ids" nolabel="1" colspan="2"
+                        readonly="True" invisible= "is_internal_transfer">
```

Comment:
```suggestion
                        readonly="True" invisible="is_internal_transfer">
```

Reply:
@maq-adhoc is this ready to be closed? As I see this is Outdated.
cc' @jco-odoo

Reply:
done